src/crystalnet.h

```diff
@@ -85,9 +67,9 @@ typedef struct s_node_t s_node_t;
 typedef s_node_t *symbol;
 typedef struct s_model_t s_model_t;
 typedef struct s_model_ctx_t s_model_ctx_t;
-extern s_model_ctx_t *new_s_model_ctx();
+extern s_model_ctx_t *make_s_model_ctx();
 extern s_model_t *new_s_model(s_model_ctx_t *, s_node_t *, s_node_t *);
-extern void free_s_model(s_model_t *);
+extern void del_s_model(s_model_t *);
```

Comment:
if we only use s_model, shall we rename them all to model? 

The same applies to layer, node ...

Reply:
The physical structs, including model, layer, node etc will still exists internally.